src/util/reviews.tsx

```diff
@@ -0,0 +1,78 @@
+// React
```

Comment:
Could you move reviews.tsx to the components folder please and capitalize the filename? util is usually for helper functions that are intended to be called from many other places in the code

Reply:
Got it!